cdap-data-fabric/src/main/java/io/cdap/cdap/store/StoreDefinition.java

```diff
@@ -91,9 +91,11 @@ private static void createIfNotExists(StructuredTableAdmin admin,
    */
   public static final class NamespaceStore {
     public static final StructuredTableId NAMESPACES = new StructuredTableId("namespaces");
+    public static final StructuredTableId REPOSITORIES = new StructuredTableId("repositories");
 
     public static final String NAMESPACE_FIELD = "namespace";
     public static final String NAMESPACE_METADATA_FIELD = "namespace_metadata";
+    public static final String REPOSITORY_CONFIGURATION_FIELD = "repository_config";
```

Comment:
nit: just name it 'config', repository is redundant since the table name is repositories